openproblems/tasks/dimensional_reduction/datasets/10x_pbmc.py

```diff
@@ -0,0 +1,8 @@
+  
+from ....data._10x_5k_pbmc import load_10x_5k_pbmc
+from ....tools.decorators import dataset
+
+
+@dataset("10X PBMCs v3 Chemistry")
+def _10x_5k_pbmc(test=False):
```

Comment:
This shouldn't be a private method -- can we remove the underscore?

Reply:
Bump -- underscore should be removed here. Also this method should be imported in datasets/__init__.py.

Reply:
fixed